pkg/apis/pipeline/v1beta1/matrix_types_test.go

```diff
@@ -143,15 +610,15 @@ func TestMatrix_HasParams(t *testing.T) {
 		{
 			name: "matrixed with params",
 			matrix: &Matrix{
-				Params: []Param{{Name: "platform", Value: ParamValue{ArrayVal: []string{"linux", "windows"}}}},
+				Params: Params{{Name: "platform", Value: ParamValue{ArrayVal: []string{"linux", "windows"}}}},
```

Comment:
this cleanup is not related to this change, and is also not applied in the v1 files

could you please remove this cleanup, and open a separate PR making this change in both v1beta1 and v1 🙏🏾

Reply:
Sorry, I thought the params had been changed already in this PR: #6407

Reply:
no worries, please open another PR cleaning up those params 🙏🏾

Reply:
@jerop Opened a PR here: https://github.com/tektoncd/pipeline/pull/6446 based off this one.